### ISSUE TITLE
Hard crash on M1 max MBP

### ISSUE THREAD

**Issue by Rob-Syndic** [AdiAddons/AdiBags#624]
Hey I'm sorry I don't have more colour on this one, but here's how to replicate:

**Wow version:** 9/1/5 (41488) (Release arm64)
**Adibags version:** 1.9.26

**Steps to replicate:** Open bags (b), or any other interaction that automatically opens bags (mailbox, vendor, bank, etc).
Result: hard crash to desktop

If I turn off the add-on in the character selection screen, I have no trouble with the bags or other bag-related interactions (mail, vendor, bank).  I have tried unloading different related add-ons (Adibags - Filter for BoE/BoA, Adibags - PT3 filter, Adibags - Shadowlands, Adibags Configuration) in an effort to isolate, but no combination of those stops the hard crash when opening bags.

I don't get any LUA error or other message... just instant crash back to desktop.

I recognize this may be difficult to test, so I'm hoping some other users are experiencing the same problems.

Thanks for a great add-on that (unfortunately), I've come to see as mandatory... especially after going to back to the Blizz version of bags over the past few weeks!

**Comment by prethesis** [comment]
Hi, 

thanks for the great add-on.

I have the same issue but only on two of my characters. AdiBags works exceptionally fine on all other characters.

WoW Version:
Adibags version: 1.9.26

Also on M1 Mac, but normal Mac Mini M1 (2020)

The configuration through the Interface menu works. I tried to configure an individual profile for the characters with the same result.

I moved the AdiBags files from WTF and I tried to edit the two characters out of the config files and caches without any luck. No guarantee I got that right in all places though I thing I was careful.

I emptied the bags and played the chars to 60 to try and rule out anything level related. 

I deactivated all other add-ons. Same effect. 

ArkInventory and Bagnon (wanted to see if some add-on interaction is causing it) work on these two characters as do the built-in bags. 

I would love to assist by providing debug or log information, but the crash is just instant and I can not seem to find any logs or exception artifacts. Anything I can do to provide more information?
